source/brailleDisplayDrivers/handyTech.py

```diff
@@ -156,6 +158,14 @@ def postInit(self):
 		not been set, which is needed for sending packets to the display.
 		"""
 
+	def _get__display(self):
+		"The L{BrailleDisplayDriver} which initialized this Model instance"
+		# self._displayRef is a weakref, call it to get the object
+		if self._displayRef and callable(self._displayRef):

```

Comment:
Why this if check? Note that calling a weak reference returns None when the object died, so you can just do 			return self._displayRef()